### PR TITLE
pidstore: external datacite package 

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

**Comment by lnielsen** [comment]
Whoops....rebased to ``maint-2.0``

File: invenio/utils/datacite.py [hunk 1]
```diff
@@ -17,563 +17,75 @@
...
-        return DataCiteRequest(
-            base_url=self.api_url,
-            username=self.username,
-            password=self.password,
-            default_params=params,
+        """Initialize API client.
+
+        Compatibility layer on top of external DataCite API client.
+        """
+        warnings.warn("Use of invenio.utils.datacite:DataCite is been "
```
Comment (lnielsen) [inline invenio/utils/datacite.py]: spelling mistake

**Comment by lnielsen** [comment]
@egabancho Can I get you to take a quick look at this?

**Comment by egabancho** [comment]
Looks ok to me! (I haven't tested it though)

**Pull request closed without merging**